packages/opds-web-client/src/components/Breadcrumbs.tsx

```diff
@@ -18,21 +18,23 @@ export default class Breadcrumbs extends React.Component<BreadcrumbsProps, {}> {
   };
   render(): JSX.Element {
     return (
-      <ol className="breadcrumb" aria-label="breadcrumbs" role="navigation">
-        { this.props.links && this.props.links.map((link, i) =>
-          <li key={link.url}>
-            {
-              (i === this.props.links.length - 1) && (!this.props.currentLink) ?
-                <span>{link.text}</span> :
-                <CatalogLink
-                  collectionUrl={link.url}
-                  bookUrl={null}>
-                  {link.text}
-                </CatalogLink>
-            }
-          </li>
-        ) }
-      </ol>
+      <nav aria-label="breadcrumbs" role="navigation">
+        <ol className="breadcrumb">
```

Comment:
It seems confusing that the `ol` rather than the `li` is called "breadcrumb"; I realize that didn't come from this PR, but I think calling the nav "breadcrumbs" makes it even more confusing.  I would go `nav.breadcrubs-nav', `ol.breadcrumbs`, `li.breadcrumb`.  But this may not be worth messing with; it's not at all a big deal.